vault.js

```diff
@@ -1,4 +1,17 @@
 'use strict';
 module.exports = function() {
-  
-};
\ No newline at end of file
+    let newKey;
```

Comment:
So the way this is, it can only hold one item in the "vault" at any point in time. It's better to use an object so that this can hold multiple items.